addons/resource/models/resource_mixin.py

```diff
@@ -64,7 +64,7 @@ def get_work_days_data(self, from_datetime, to_datetime, calendar=None, domain=N
             work_time = sum((interval[1] - interval[0] for interval in day_intervals), timedelta())
             total_work_time += work_time
             if theoric_hours:
-                days_count += float_utils.round((work_time.total_seconds() / 3600 / theoric_hours) * 4) / 4
+                days_count += float_utils.round((work_time.total_seconds() / 3600 / theoric_hours) * 8) / 8
```

Comment:
What is the reason of this change ? I know we talked about it but I don't remember. Maybe it could be done in a first separated commit with its own explanation, just to separate a bit things.

Reply:
The problem is that previously we were forcing to count in quarters of days, but we are now taking leaves in hours which makes it problematic.

I've chosen a precision of 8 arbitrarily (as was the case before), but we could discuss this choice

ps: The changes to the tests reflect that change of precision